src/consumer.ts

```
@@ -253,7 +283,14 @@ export class Consumer extends EventEmitter {
     try {
       return await this.sqs.receiveMessage(params).promise();
     } catch (err) {
-      throw toSQSError(err, `SQS receive message failed: ${err.message}`);
+      const sqsError = toSQSError(
+        err,
+        `SQS receive message failed: ${err.message}`
+      );
+
+      if (!isAbortedError(sqsError)) {
+        throw sqsError;
```

Do we not want to throw this error still? I think we probably do...